src/main/java/utils/UI.java

```diff
@@ -0,0 +1,21 @@
+package utils;
+
+public class UI {
```

Comment:
Should UI class contain only the greet and signoff print? I think it is better if all the UI related methods are inside this class instead.